content/blog/2020/01/2020-01-27-jenkins-world-lisbon-with-love-from-india.adoc

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Jenkins World Lisbon Experience"
+tags:
+- gitlab
+- lisbon
+- devopsworld
+- jenkinsworld
+- gsoc
+- gsoc2019
+author: baymac
+description: "Experience of attending Jenkins World Lisbon 2019"
+opengraph:
+  image: /images/post-images/2020/01-gsoc-jenkins-world-parichay/project-presentation.jpg
+---
+
+= My personal experience of Jenkins World Lisbon
+
+After an amazing three months of development period in the summers of 2019 with Jenkins Project, I was a better developer, loved open source, met
+passionate people and had fun at work. Jenkins is not just a community, it is a family. When GSoC period was over, we received swags from Jenkins. 
+Natasha Stopa (one of the GSoC 19 student) was invited to attend Jenkins World San Fransisco. It was nice to see her enjoy there. But guess what? Jenkins 
```

Comment:
This `Natasha Stopa (one of the GSoC 19 student)` should be `Natasha Stopa (one of the GSoC 19 students)`

Reply:
Looks too much like there were 19 students, rather than "one of the students in GSoC 2019".